hanami-utils.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.0'
 
   spec.add_dependency 'transproc', '~> 1.0'
+  spec.add_dependency 'wisper', '~> 2.0.0'
```

Comment:
just a question: could we drop wisper dependency? Or make it not require dependency for hanami-utils?